.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Use this template for submitting bug reports.
+title: "[BUG]: [ISSUE TITLE]"
+labels: bug
+assignees:
+---
+
+<!--
+Thank you for taking the time to create an issue for your topic/question.
+This issue provides a permanent record for others with similar questions
+to gain insight at a later date - your contributions here are significant
+to the community and much appreciated!
+
+Please always remember to be courteous to others posting on this forum as
+we are all here voluntarily to help others use Isca.
+
+NOTE:
+Isca team members are all full-time research scientists, so while we are very happy to provide
+some support, response times may vary. If you need in-depth support, then we may suggest that one
+ of us becomes a co-author on your next paper by providing a long-term support on the issue.
+-->
+
+<!--
+Please be sure to add an informative title to this issue
+-->
+
+## Description
+<!--
+Fill out a concise description of your topic/question in this section.
+Feel free to post your code here using Markdown formatting
+-->
+
+-----
+**Screenshots**
+<!--
+If applicable, add screenshots to help explain your problem.
+-->
+
+-----
+**System Information:**
```

Comment:
Perhaps we could also ask about their most recent commit ID / are they working off their own fork or just using the ExeClim one?

Reply:
yes, good point!